src/hotspot/share/classfile/classLoader.cpp

```diff
@@ -1295,7 +1296,7 @@ char* ClassLoader::skip_uri_protocol(char* source) {
 
 // Record the shared classpath index and loader type for classes loaded
 // by the builtin loaders at dump time.
-void ClassLoader::record_result(InstanceKlass* ik, const ClassFileStream* stream, TRAPS) {
+void ClassLoader::record_result(Thread* current, InstanceKlass* ik, const ClassFileStream* stream) {
```

Comment:
So here you converted TRAPS to "Thread* current" but you didn't in the above functions - why not? This is inconsistent and seems arbitrary.

Reply:
Please refer to my reply above. Hopefully, the latest change is more consistent.